server/secops/secops_mcp/tools/security_rules.py

```diff
@@ -131,6 +131,87 @@ async def search_security_rules(
         logger.error(f'Error searching security rules: {str(e)}', exc_info=True)
         return {'error': str(e), 'rules': []}
 
+@server.tool()
+async def get_detection_rule(
+    rule_id: str,
+    project_id: str = None,
+    customer_id: str = None,
+    region: str = None,
```

Comment:
I found some clients don't recognize the params when `str = None` is used and they then prompt the user to supply them. It seems that `Optional[str] = None,` fixes this issue. So, could you please make that change here?